docker-images/flask-only/requirements.txt

```diff
@@ -0,0 +1 @@
+flask
```

Comment:
As a suggestion - to make sure that all tests will be running the same software.  You may want to pin the requirements to a specific version:

`$ pip freeze > requirements.txt`